internal/pkg/cli/deploy/job_test.go

```diff
@@ -0,0 +1,4 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
should we delete these empty test files? :rip:

Reply:
i think it's kinda sus that they are empty, so i left them.... 🤔

Reply:
(as in, tests on `WorkloadDeployer` should probably be on these types)